templates/BUILD.bazel

```
@@ -24,6 +24,7 @@ genrule(
             mkdir -p $$OUT/$$d
             cp -rL $$SRC/$$d/* $$OUT/$$d/
             cp $$SRC/default-gitignore $$OUT/$$d/.gitignore
+            cp $$SRC/default-daml-yaml $$OUT/$$d/.dlint.yaml
```

```suggestion
            cp $$SRC/default-dlint-yaml $$OUT/$$d/.dlint.yaml
```
I believe you also need to add it to `srcs`?